### PR TITLE
DO-1044 Header+footer logos

### PR DESCRIPTION
**Pull request by Litsyshyn**
Changed logos according to design requirments.
https://www.figma.com/file/2nlc5QYVVpr0pawtFvTSiD/Landing-PEP-Data-Ocean?node-id=1087%3A355

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /.env
 /.vscode
 /.idea
+/.env.local
```
Comment (roman-tiukh) [inline .gitignore:6]: /.env.local is not exists in this project

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
1. `.env.example → .env.local`
why you renamed this?

File: .gitignore (hunk 1 above)
Comment (Litsyshyn) [reply .gitignore:6]: My bad.

**Comment by Litsyshyn** [comment]
Closed because i wrong understand the task.

**Pull request closed without merging**